client/src/App.js

```diff
@@ -50,8 +51,8 @@ class App extends Component {
               path="/places/:id"
               render={props => (
                 <OnePlace
-                  title="Props through render"
-                  place={this.state.listOfPlaces[props.match.params.id]}
+                  title={`Props through render`}
```

Comment:
What does this mean?

Reply:
In this file I jus adding thigs for Arabic List words and login